src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/hover/PropertiesHoverParticipant.java

```diff
@@ -211,34 +222,30 @@ protected boolean isAdaptableFor(PsiElement hoverElement) {
 
 	/**
 	 * Returns all the config property information for the given property key.
-	 *
+	 * <p>
 	 * Includes the information for all the different profiles.
 	 *
-	 * @param propertyKey                      the property key without the profile
-	 * @param annotation                       the annotation that defines the
-	 *                                         config property
-	 * @param defaultValueAnnotationMemberName the annotation member name for
-	 *                                         default value and null otherwise.
-	 * @param project                          the project
-	 * @param utils                            the JDT LS utilities.
+	 * @param propertyKey  the property key without the profile
+	 * @param annotation   the annotation that defines the
+	 *                     config property
+	 * @param defaultValue default value and null otherwise.
+	 * @param project      the project
+	 * @param utils        the JDT LS utilities.
 	 * @return the config property information for the given property key
 	 */
 	private static List<MicroProfileConfigPropertyInformation> getConfigPropertyInformation(String propertyKey,
-			PsiAnnotation annotation, String defaultValueAnnotationMemberName, PsiFile typeRoot,
-			PsiMicroProfileProject project, IPsiUtils utils) {
+																							PsiAnnotation annotation,
+																							String defaultValue,
+																							PsiFile typeRoot,
+																							PsiMicroProfileProject project,
+																							IPsiUtils utils) {
```

Comment:
what happened with the formatting here?

Reply:
When I replaced this file with Quarkus's file  this custom lines breaks came.